publication-commons/src/main/java/no/unit/nva/publication/service/impl/MessageService.java

```diff
@@ -75,30 +75,30 @@ public static SortableIdentifier extractIdentifier(URI uri) {
 
     @JacocoGenerated
     @Deprecated
-    public URI createMessage(UserInstance sender,
-                             Publication publication,
-                             String messageText) throws TransactionFailedException {
+    public SortableIdentifier createMessage(UserInstance sender,
```

Comment:
The service does not need to return a URI. The DTO is constructed by the handler.